gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/Option.java

```diff
@@ -69,4 +73,20 @@ public String toString() {
         .add("value", value)
         .toString();
   }
+
+  static String selector(List<FieldSelector> required, FieldSelector... others) {
```

Comment:
Any reason (considering we expose the `selector` method - which I think is fine)?